### PR TITLE
docs(contributing): commands to install jest globally

### PR DESCRIPTION
**Pull request by anshumanv**
Add commands to install jest globally for running individual tests

ISSUES CLOSED: #775

**What kind of change does this PR introduce?**
Docs

**Did you add tests for your changes?**
NA

**If relevant, did you update the documentation?**
Yes

**Summary**
Added commands to install jest cli for running tests.

**Does this PR introduce a breaking change?**
No

**Other information**
NA

### CODE REVIEW

**Comment by anshumanv** [comment]
@PlayMa256 updated, looks better?

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -49,7 +49,9 @@ npm link webpack-cli
   - `BIN_TEST_CASES_GREP=/myCase jest test/BinTestCases.test.js`
 
 * To test a single CLI (other type of) test case:
-  - `jest path/to/my-test.js` 
+  - `npx jest path/to/my-test.js` 
+  - You can also install jest globally and run tests without npx.
+    - `npm i -g jest`
```
Comment (rishabh3112) [inline .github/CONTRIBUTING.md]:
```suggestion
    - `jest path/to/my-test.js`
```
Add command to run test when jest is installed globally.

Comment (anshumanv) [reply .github/CONTRIBUTING.md]: Done 👍

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -49,7 +49,11 @@ npm link webpack-cli
   - `BIN_TEST_CASES_GREP=/myCase jest test/BinTestCases.test.js`
 
 * To test a single CLI (other type of) test case:
-  - `jest path/to/my-test.js` 
+  - `npx jest path/to/my-test.js` 
+  - You can also install jest globally and run tests without npx.
```
Comment (evenstensberg) [inline .github/CONTRIBUTING.md]: `:` at the end of paragraphs here.

Comment (anshumanv) [reply .github/CONTRIBUTING.md]: Thanks, Missed that!

Fixed. 👍

**Pull request merged**